repo-agent/review-ui/review-api/pkg/api/handlers_pr.go

```diff
@@ -293,18 +243,18 @@ func (s *Server) submitReview(c *gin.Context) {
 	}
 	log.Printf("review created: %v", review)
 	// Set review in Redis
-	err = s.Redis.HSet(c.Request.Context(), prKey, "review", payload.Review).Err()
+	err = s.Store.UpdatePRReview(c.Request.Context(), namespace, repo, prID, payload.Review)
 	if err != nil {
 		c.JSON(http.StatusInternalServerError, gin.H{"error": "Failed to save review", "details": err.Error()})
 		return
 	}
 
 	// Delete draft from Redis
-	err = s.Redis.HSet(c.Request.Context(), prKey, "draft", "").Err()
-	if err != nil {
-		c.JSON(http.StatusInternalServerError, gin.H{"error": "Failed to clear draft", "details": err.Error()})
-		return
-	}
+	//err = s.Redis.HSet(c.Request.Context(), prKey, "draft", "").Err()
```

Comment:
Same question here - maybe needs a comment describing why it is commented out?

Reply:
removed.